Assets/HoloToolkit/Utilities/Scripts/DebugPanelControllerInfo.cs

```diff
@@ -94,6 +123,7 @@ private string GetControllerInfo()
             string toReturn = "";
             foreach (ControllerState controllerState in controllers.Values)
             {
+                // Debug message
                 toReturn += string.Format("Hand: {0}\nPointer: Position: {1} Rotation: {2}\n" +
```

Comment:
This still shows up, causing duplicate info across the text panels (though, interestingly, I don't see it in any of your screenshots). Should we remove it in favor of yours?

Reply:
Daft question @cre8ivepark .  Why is this all using the older "TextMesh" components rather than the new UI system?  Just seems odd.

Reply:
@DDReaper really either can be used. It makes no difference.

Reply:
It's more of a performance issue.